src/tests/unit/connection.test.js

```diff
@@ -14,14 +13,12 @@ chai.use(chaiAsPromised);
 chai.use(sinonChai);
 
 describe('Test oracledb connection module', () => {
-  let configGetStub;
   let connection;
 
-  beforeEach(() => {
-    configGetStub = sinon.stub(config, 'get')
-      .withArgs('dataSources.oracledb')
-      .returns({});
-  });
+  const configGetStub = sinon.stub()
```

Comment:
Hmm, why don't we need to stub `config` anymore? I believe we have to do it for other APIs? Is it any behavior changed for persons v2?

Reply:
It is being stubbed with proxyquire https://github.com/osu-mist/express-api-skeleton/blob/9caf304a89100840bda413cd0528d2724f2e57d6/src/tests/unit/connection.test.js#L26. The reason we didn't see this before is because the connection tests were running before other tests. It was the `address-dao` tests that exposed this because the tests run in alphabetical order.

Reply:
ok, make sense.